tests/tests_templates/test_python_templates.py

```diff
@@ -280,6 +280,7 @@
 # Structure for solution projects
 SOLUTION_STRUCTURE = [
     ".github/workflows/build-release.yml",
+    ".github/workflows/release-please.yml",
```

Comment:
There are 2 additional config files that need to be included in order for this workflow to run successfully: https://github.com/ansys-internal/volitare/commit/6cdd5ea7d937a07106e4d4c5b25ae6a26bd786fb

Reply:
Added.